src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6592,6 +6592,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
   <data name="ERR_RecordAmbigCtor" xml:space="preserve">
-    <value>A record cannot be declared with a single positional parameter with the type of that record.</value>
+    <value>Synthesized copy constructor conflicts with a primary constructor</value>
```

Comment:
Should end with a period?

Reply:
We're not very consistent on that, but sure

---
In reply to: [543645195](https://github.com/dotnet/roslyn/pull/49975#discussion_r543645195) [](ancestors = 543645195)